drivers/base/bootvid/common.c

```diff
@@ -15,8 +15,59 @@ ULONG VidpScrollRegion[4] =
     SCREEN_HEIGHT - 1
 };
 
+PALETTE_ENTRY DefaultPalette[MAX_COLORS] =
+{
+    {0x00, 0x00, 0x00}, // Black
+    {0x00, 0x00, 0x80}, // Blue
+    {0x00, 0x80, 0x00}, // Green
+    {0x00, 0x80, 0x80}, // Cyan
+    {0x80, 0x00, 0x00}, // Red
+    {0x80, 0x00, 0x80}, // Magenta
+    {0x80, 0x80, 0x00}, // Brown
+    {0xC0, 0xC0, 0xC0}, // Light Gray
+    {0x80, 0x80, 0x80}, // Dark Gray
+    {0x00, 0x00, 0xFF}, // Light Blue
+    {0x00, 0xFF, 0x00}, // Light Green
+    {0x00, 0xFF, 0xFF}, // Light Cyan
+    {0xFF, 0x00, 0x00}, // Light Red
+    {0xFF, 0x00, 0xFF}, // Light Magenta
+    {0xFF, 0xFF, 0x00}, // Yellow
+    {0xFF, 0xFF, 0xFF}, // White
+};
+
 /* PRIVATE FUNCTIONS **********************************************************/
 
+VOID
+NTAPI
+InitializePalette(VOID)
+{
+    ULONG i, pal;
+
+    /* Loop all the entries and set their palettes */
+    for (i = 0; i < MAX_COLORS; i++)
+    {
+        pal = (DefaultPalette[i].Red << 16) | (DefaultPalette[i].Green << 8) | DefaultPalette[i].Blue;
+        SetPaletteEntry(i, pal);
+    }
+}
+
+VOID
+NTAPI
+InitPaletteWithTable(
```

Comment:
And here:
https://git.reactos.org/?p=reactos.git;a=blob;f=drivers/base/bootvid/arm/bootvid.c;hb=592bafd60a33b82e62174f124ce80b2e0374a03a#l168

Reply:
Fixed.